terraform/modules/cloudwatch/alarms.tf

```diff
@@ -0,0 +1,98 @@
+module "aws-rds-alarms" {
+  source                               = "lorenzoaiello/rds-alarms/aws"
+  version                              = "2.2.0"
+  db_instance_id                       = var.db_instance_id
+  db_instance_class                    = var.db_instance_class
+  disk_burst_balance_too_low_threshold = var.db_low_disk_burst_balance_threshold
+  actions_alarm                        = var.enable_alerts == true ? [var.rds_db_alerts_topic_arn] : []
+  actions_ok                           = var.enable_alerts == true ? [var.rds_db_alerts_topic_arn] : []
+}
+
+module "backoffice-aws-alb-alarms" {
+  source           = "lorenzoaiello/alb-alarms/aws"
+  version          = "1.2.0"
+  load_balancer_id = var.backoffice_alb_id
+  prefix           = "elb-${var.backoffice_load_balancer}"
+  target_group_id  = var.backoffice_alb_target_group_id
+  actions_alarm    = var.enable_alerts == true ? [var.backoffice_lb_alerts_topic_arn] : []
+  actions_ok       = var.enable_alerts == true ? [var.backoffice_lb_alerts_topic_arn] : []
+}
+
+module "webapp-aws-alb-alarms" {
+  source           = "lorenzoaiello/alb-alarms/aws"
+  version          = "1.2.0"
+  load_balancer_id = var.webapp_alb_id
+  prefix           = "elb-${var.webapp_load_balancer}"
+  target_group_id  = var.backoffice_alb_target_group_id
+  actions_alarm    = var.enable_alerts == true ? [var.webapp_lb_alerts_topic_arn] : []
+  actions_ok       = var.enable_alerts == true ? [var.webapp_lb_alerts_topic_arn] : []
+}
+
+module "backoffice_ecs_service_alarms" {
+  source = "cloudposse/ecs-cloudwatch-sns-alarms/aws"
+  # Cloud Posse recommends pinning every module to a specific version
+  version                               = "0.12.3"
+  namespace                             = "ecs"
+  stage                                 = terraform.workspace
+  name                                  = "droits-${var.ecs_backoffice_service_name}"
+  cluster_name                          = var.ecs_cluster_name
+  service_name                          = var.ecs_backoffice_service_name
+  cpu_utilization_high_alarm_actions    = var.enable_alerts == true ? [var.ecs_backoffice_alerts_topic_arn] : []
+  cpu_utilization_high_ok_actions       = var.enable_alerts == true ? [var.ecs_backoffice_alerts_topic_arn] : []
+  memory_utilization_high_alarm_actions = var.enable_alerts == true ? [var.ecs_backoffice_alerts_topic_arn] : []
+  memory_utilization_high_ok_actions    = var.enable_alerts == true ? [var.ecs_backoffice_alerts_topic_arn] : []
+}
+
+resource "aws_cloudwatch_metric_alarm" "backoffice_ecs_service_task_count_too_low" {
+  alarm_name          = "ecs-${terraform.workspace}-droits-${var.ecs_backoffice_service_name}-low-task-count"
+  comparison_operator = "LessThanThreshold"
+  evaluation_periods  = 1
+  metric_name         = "CPUUtilization"
+  namespace           = "AWS/ECS"
+  period              = 60
```

Comment:
This should probably be a var

Reply:
Which bit are you referring to specifically? Is it evaluation_periods?

Reply:
Yeah, basically anything that's configurable should have sensible names, and be configurable from the vars file. 

For instance:

In dev, it's fine if it's running at 95%
In prod, we want it more at like 70-80%

So these can be interchangeable ver env from the tfvars file.

Does this make sense?